terraform/parser.go

```diff
@@ -90,7 +90,7 @@ func (p *DefaultParser) Parse(body string) ParseResult {
 func (p *FmtParser) Parse(body string) ParseResult {
 	result := ParseResult{}
 	if p.Fail.MatchString(body) {
-		result.Result = "There is diff in your .tf file (need to be formatted)"
+		result.Result = body
```

Comment:
I think it's ok to leave the result string as is. Instead, we should modify the template to also collapse the body (as you did in the PR), re-add `.Message` and `.Result` and modify [these lines](https://github.com/mercari/tfnotify/blob/4fc11707df04531d6675df2fcc657dc5315fb1d7/terraform/template.go#L244-L245) to set body properly.

Reply:
That makes so much sense! thanks for the feedback!